src/components/Config/Config.tsx

```diff
@@ -120,68 +146,148 @@ function Config() {
+          >
+            <Radio.Button value={1} className="radiobuttons">
+              <img alt="eth-network-icon" src="/assets/eth.svg" width="32" height="32" className="network-icon" />
+              Ethereum
+            </Radio.Button>
+            <Radio.Button value={2}>
+              <img alt="eth-network-icon" src="/assets/arbitrum.svg" width="32" height="32" className="network-icon" />
+              Arbitrum
+            </Radio.Button>
+            {/* if we want to add more networks.
+              <Radio value={3}>C</Radio>
+              <Radio value={4}>D</Radio>
+            */}
+          </Radio.Group>
+
+          {networkvalue < 2 && (
+            <>
+              <Checkboxes>
+                <CheckboxSection>
+                  <Checkbox
+                    checked={gasPriceAware}
+                    disabled={submitted}
+                    onChange={(e) => setGasPriceAware(e.target.checked)}
+                  >
+                    <Typography.Paragraph className="subtitle">Gas price aware</Typography.Paragraph>
+                  </Checkbox>
+                  <p>
+                    Once the network gas price falls below the gas cost specified in the transaction, it will be
+                    broadcast to the network.
+                  </p>
+                </CheckboxSection>
+                <CheckboxSection>
+                  <Checkbox checked={draft} disabled={submitted} onChange={(e) => setDraft(e.target.checked)}>
+                    <Typography.Paragraph className="subtitle">Draft mode (Advanced)</Typography.Paragraph>
+                  </Checkbox>
+                  <p>
+                    No transaction will be broadcast to the Ethereum network until you go to the Transaction list and
+                    specify additional conditions.
+                  </p>
+                </CheckboxSection>
+              </Checkboxes>
+              <div>
+                <Typography.Paragraph className="subtitle">Estimated Confirmation Time</Typography.Paragraph>
+                <p>
+                  Our algorithms analyze historical gas prices in real time to best propose a gas price for you. Longer
+                  wait times generally correspond to cheaper gas prices. Since we cannot control the Ethereum network,
+                  this is NOT a guaranteed confirmation time.
+                </p>
+                <SliderContainer>
+                  <Slider
+                    marks={sliderMarks}
+                    step={1}
+                    value={confirmationTime}
+                    min={0}
+                    max={3}
+                    disabled={submitted}
+                    tooltipVisible={false}
+                    onChange={setConfirmationTime}
+                  />
+                </SliderContainer>
+              </div>
+            </>
+          )}
+          <Button type="primary" size="large" onClick={() => handleConnect(networkvalue < 2 ? 'Ethereum' : 'Arbitrum')}>
```

Comment:
fix network check. Also capitalize correctly - `networkValue`. Might also want to rename to be more precise: `networkId` or `networkName` etc depending on what the value really is